openedx/stanford/cms/djangoapps/contentstore/tests/test_utilities.py

```diff
@@ -37,6 +37,7 @@ def compare_utilities(self, persisted, request):
 
     def test_get_utilities(self):
         """ Tests the get utilities method and URL expansion. """
+        self.assertTrue(False)
```

Comment:
@caesar2164 Yeah, that's why it's in a separate commit that explicitly mentions it fails on purpose ;)